### PR TITLE
List

### PR DESCRIPTION
**Pull request by arundhatij12**

### CODE REVIEW

File: list.css [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+}
+
+.content2 {
+  display: flex;
+  gap: 10px;
+  padding: 3px;
+  width: 800px;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
```
Comment (srejitk) [inline list.css:81]: Would suggesting making use of relatives units and utility classes

File: input.css [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+  flex-wrap: wrap;
+  height: max-content;
+  margin-top: 50px;
+}
+
+.content1 {
+  margin-left: 115px;
+  align-items: center;
+  justify-content: center;
+}
```
Comment (srejitk) [inline input.css:48]: Use appropriate class names to prevent confusion.

**Review by srejitk — APPROVED** [review]
Suggested a few changes. Looks good otherwise.

File: input.css [hunk 3]
```diff
@@ -0,0 +1,185 @@
+body {
+  background: linear-gradient(to bottom, #66ffcc 0%, #66ccff 105%);
+  font-family: "Gill Sans", "Gill Sans MT", Calibri, "Trebuchet MS", sans-serif;
+}
+
+header {
+  background-color: rgba(114, 195, 202, 0.575);
```
Comment (rahulcs754) [inline input.css:7]: provide colour variable

File: input.css [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+  height: 80px;
+}
+
+a {
+  text-decoration: none;
+  color: #3d6dc5;
```
Comment (rahulcs754) [inline input.css:13]: provide colour variable

**Review by rahulcs754 — APPROVED** [review]
try to remove extra space 
create color variable 
try to create utility class for flex center  flex row

**Pull request merged**